criu/cgroup.c

```diff
@@ -1906,6 +1906,8 @@ static int rewrite_cgsets(CgroupEntry *cge, char **controllers, int n_controller
 						    cg->path + dirlen + 1);
 				if (!cg->path) {
 					cg->path = prev;
+					if (dirnew)
+						xfree(dirnew);
```

Comment:
xfree can handler NULL, so you don't need if (dirnew) here

Reply:
removed if